src/organisms/news/datasource/index.js

```diff
@@ -0,0 +1,16 @@
+import Prismic from "@prismicio/client"
+
+const apiEndpoint = "https://netdata-news.cdn.prismic.io/api/v2"
+const client = Prismic.client(apiEndpoint)
+
+const noop = () => {}
+
+export const fetchNews = (app, onSuccess = noop, onError = noop) => {
+  return client
+    .query(Prismic.Predicates.at("document.tags", [app]), {
+      pageSize: 100,
+      orderings: "[document.last_publication_date desc]",
+    })
+    .then(onSuccess)
+    .catch(onError)
```

Comment:
I think that the `noop` is not needed, the promise chaining handles it correctly.